windows/src/buildtools/sentrytool/Keyman.System.SentryTool.SentryToolMain.pas

```diff
@@ -20,31 +20,158 @@ implementation
   FInfile, FOutfile, FProjectFile, FRootPath, FIncludePath: string;
   FDelphiSearchFile: TDelphiSearchFile = nil;
 
+  CmdParams: array of string;
+
+// Note: this is lifted from System.pas, which doesn't
+// make GetParamStr a public symbol, sadly.
```

Comment:
We probably can't do this... so I may need to rewrite this silly little function

Reply:
Is this PR ready to review, or is the rewrite still TODO?

Reply:
I'm returning this to DRAFT until such time as I rewrite this silly function (probably 15.0 alpha)